Resources/Prototypes/Entities/Mobs/NPCs/revenant.yml

```diff
@@ -62,7 +62,6 @@
     interfaces:
     - key: enum.StoreUiKey.Key
       type: StoreBoundUserInterface
-  - type: MobState
   - type: Visibility
```

Comment:
No mob thresholds?

Reply:
Revenant cannot die, even on master. Their mobState component doesn't have any states in it. With the new system that can result in issues